scss/_maps.scss

```diff
@@ -45,18 +45,6 @@ $theme-colors-border-subtle: (
 ) !default;
 // scss-docs-end theme-border-subtle-map
 
-// Boosted mod
-$background-colors: map-merge(
```

Comment:
I'm not sure it's been mentioned in the migration guide. Please double-check all deprecated/new Sass/CSS variables from this PR so that the migration guide is complete.